tests/PySys/pysysproject.xml

```diff
@@ -7,7 +7,7 @@
 	<property name="tebasedir" value="${env.TEBASEDIR}" default="${testRootDir}/../.."/>
 	<property name="exampledir" value="${env.EXAMPLEDIR}" default="${testRootDir}/../../ci"/>
 	<property name="c8yurl" value="${env.C8YURL}" default=""/>
-	<property name="username" value="${env.C8YUSERNAME}" default=""/>
+	<property name="c8yusername" value="${env.C8YUSERNAME}" default=""/>
 	<property name="c8ypass" value="${env.C8YPASS}" default=""/>
 	<property name="tenant" value="${env.C8YTENANT}" default=""/>
 	<property name="deviceid" value="${env.C8YDEVICEID}" default=""/>
```

Comment:
While you're at renaming things, do you wanna rename this `deviceid` to `c8ydeviceid` as well? And may be rename `device` to `c8ydevicename`?

Reply:
So far, it was just the bare minimum to get it working with the new pysys version. I would propose that we change the names with the next refactoring of the code. I've seen other things while browsing through the code.

Reply:
Btw, why did you have to change? Is there some internal field with the same name?